Userland/Libraries/LibWeb/Layout/GridFormattingContext.cpp

```diff
@@ -1340,7 +1340,7 @@ void GridFormattingContext::stretch_auto_tracks(GridDimension const dimension)
     }
 
     CSSPixels remaining_space = get_free_space(*m_available_space, dimension).to_px_or_zero();
-    auto remaining_space_to_distribute_per_track = remaining_space / count_of_auto_max_sizing_tracks;
+    auto remaining_space_to_distribute_per_track = count_of_auto_max_sizing_tracks ? remaining_space / count_of_auto_max_sizing_tracks : remaining_space;
```

Comment:
if `count_of_auto_max_sizing_tracks` is zero then we can simplify early return from this function :)